performance-test/tasks/user_tasks.py

```
@@ -0,0 +1,48 @@
+      headers=self.parent.get_headers(),
+      name="PATCH /api/users/{userId}"
+    )
+
+  # 회원가입 - 소량만 (bcrypt 인코딩 무거움)
+  @task(1)
+  def register(self):
+    self.client.post(
+      "/api/users",
+      json={
+        "email": f"loadtest_{uuid.uuid4().hex[:8]}@monew.com",
+        "nickname": f"테스터{random.randint(1, 999)}",
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)